### DISCUSSION TITLE
Actions plan with self-runner

### DISCUSSION THREAD

**Discussion by wmariuss** [community/community#26803] [Actions]
<p>I can not see a clear message/text on github documentation regarding the Github actions plan if i am using self-runner. If free if i use self-runner or i have 2000 min/month n case for free plan?</p>

**Comment by airtower-luna** [comment]
<p>See <a href="https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions#about-billing-for-github-actions">About billing for GitHub Actions</a> (emphasis added):</p>
<blockquote>
<p>GitHub Actions usage is free for both public repositories <strong>and self-hosted runners</strong>.</p>
</blockquote>
<p>Makes sense if you ask me, after all with self-hosted runners you’re the one who has to pay for the runner infrastructure. :slightly_smiling_face:</p>

**Comment by wmariuss** [comment, marked as answer]
<p>Yes, they are free, with not quota limitation. Just talked with Support Team.</p>
